scout/constants/clnsig.py

```diff
@@ -15,9 +15,11 @@
     13: "association not found",
     255: "other",
 }
+
 REV_CLINSIG_MAP = {
     "other": 255,
     "not_provided": 1,
+    "no_assertion_criteria_provided": 1,
```

Comment:
Hold on, isn't that supposed to be a REVSTAT rather? I don't think you will find it in CLINSIG!

Reply:
Ok but then it should be safe to remove it from here!

Reply:
Yes, I think so!

Reply:
You are right, I don't know why I added it here! Gone now!